etl/steps/data/meadow/oecd/2023-05-18/co2_air_transport.py

```diff
@@ -0,0 +1,66 @@
+"""Load a snapshot and create a meadow dataset."""
+
+import pandas as pd
+from owid.catalog import Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+from etl.snapshot import Snapshot
+
+# Initialize logger.
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("co2_air_transport.start")
+
+    # Load inputs.
+    # Load the snapshot
+    snap: Snapshot = paths.load_dependency("co2_air_transport.csv")
+
+    # Load data from the snapshot into a pandas DataFrame
+    df = pd.read_csv(snap.path, low_memory=False)
+
+    # Specify columns to use and their new names
+    cos_to_use = ["Country", "FLIGHT", "Frequency", "SOURCE", "TIME", "Value"]
+    cos_to_name = [
+        "country",
+        "flight_type",
+        "frequency",
+        "emission_source",
+        "year",
+        "value",
+    ]  # clarify what this columns mean
+
+    # Slice the DataFrame to keep only the columns of interest
+    df = df[cos_to_use]
+
+    # Rename the columns in the DataFrame
+    df = df.rename(columns=dict(zip(df.columns, cos_to_name)))
```

Comment:
Ultra nitpick

This could be also
```python
rename_cols = {
        "Country": "country",
        "FLIGHT": "flight_type",
        "Frequency": "frequency",
        "SOURCE": "emission_source",
        "TIME": "year",
        "Value": "value",
    }
df = df.rename(columns=rename_cols)[rename_cols.values()]
```